node/resolvers/repository/CostCenterRepository.ts

```diff
@@ -0,0 +1,71 @@
+import type { CostCenterInput, Result } from '../../typings'
+import {
+  COST_CENTER_DATA_ENTITY,
+  COST_CENTER_SCHEMA_VERSION,
+} from '../../mdSchema'
+import MarketingTags from '../Mutations/MarketingTags'
+
+const CostCenterRepository = {
+  saveCostCenter: async (
+    _: void,
+    organizationId: string,
+    {
+      addresses,
+      id,
+      name,
+      paymentTerms,
+      phoneNumber,
+      businessDocument,
+      stateRegistration,
+      customFields,
+      sellers,
+      marketingTags,
+    }: CostCenterInput,
+    ctx: Context
+  ): Promise<Result> => {
+    const {
+      clients: { masterdata },
+      vtex: { logger },
+    } = ctx
+
+    const costCenter = {
+      addresses,
+      id,
```

Comment:
same question about the `id` here (what happens if it is empty?)

Reply:
The masterdata never creates the empty ID, in my tests, if it is undefined, null or empty, the ID is generated. Do you think it is necessary to validate the ID and doesn't create if not informed?

Reply:
I see! That is fine then, just wanted to make sure we are not breaking the old APIs. If masterdata is working as you mentioned, I don't think we need to add any validation.